wiki/History_of_osu!/2016/en.md

```diff
@@ -143,40 +143,36 @@ In its [announcement newspost](https://osu.ppy.sh/home/news/2016-10-17-here-come
-6. [osu!weekly #48](https://osu.ppy.sh/home/news/2016-02-16-osuweekly-48)
-7. [osu!go not worth it?](https://osu.ppy.sh/community/forums/topics/425108)
-8. [2016-02 MEETING NOTES](https://blog.ppy.sh/post/140259300353/2016-02-meeting-notes)
-9. [Modding v2: Modding Panel - Feedback Thread](https://osu.ppy.sh/community/forums/topics/442285)
-10. [osu!weekly #57](https://osu.ppy.sh/home/news/2016-04-20-osuweekly-57)
-11. [15 April 2016 changelog](https://osu.ppy.sh/home/changelog/cuttingedge/20160415)
-12. [osu! | How medals looks like? (Animations) | Achievements 20/04/2016](https://www.youtube.com/watch?v=sdGe-Gv38Yw)
-13. [Meet Maria - osu!mania's new mascot!](https://osu.ppy.sh/home/news/2016-04-20-meet-maria-osumanias-new-mascot)
-14. [Changes to the Quality Assurance Team](https://osu.ppy.sh/community/forums/topics/447417)
-15. [osu! "circles!" Remix Contest](https://osu.ppy.sh/home/news/2016-05-06-osu-circles-remix-contest)
-16. [EARLY-BIRD CONTRIBUTORS](https://blog.ppy.sh/post/144539340703/early-bird-contributors)
-17. [31 May 2016 (18:22 UTC) Tweet](https://twitter.com/ppy/status/737529593184149504)
-18. [Mapping With Rewards #1 - Bounties Complete!](https://osu.ppy.sh/home/news/2016-06-01-mapping-with-rewards-1-bounties-complete)
-19. [9 June 2016 (3:18 PM UTC)) Tweet](https://twitter.com/ppy/status/741107246965235713)
-20. [osu!weekly #65](https://osu.ppy.sh/home/news/2016-06-24-osuweekly-65)
-21. [osu! mascot design contest](https://osu.ppy.sh/home/news/2016-07-18-osu-mascot-design-contest)
-22. [osu!weekly #70](https://osu.ppy.sh/home/news/2016-08-07-osuweekly-70)
-23. [New Hush-Hush Medals!](https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals)
-24. [osu!remix contest #1: Results](https://osu.ppy.sh/home/news/2016-08-29-osuremix-contest-1-results)
-25. [osu!taiko mascot voting](https://osu.ppy.sh/home/news/2016-08-22-osutaiko-mascot-voting)
-26. [osu!taiko Mascot Community Vote Results](https://osu.ppy.sh/home/news/2016-09-21-osutaiko-mascot-community-vote-results)
-27. [GMT Applications Now Open](https://osu.ppy.sh/home/news/2016-10-15-gmt-applications-now-open)
-28. [Here Comes the Love](https://osu.ppy.sh/home/news/2016-10-17-here-comes-the-love)
-29. [Show Your Love!](https://osu.ppy.sh/home/news/2016-10-21-show-your-love)
-
-[r]: #references
+[^osu-weekly-47]: [News post by Nyquill (2016-02-09) "osu!weekly #47"](https://osu.ppy.sh/home/news/2016-02-09-osuweekly-47)
+[^jan-28-changelog]: [Cutting Edge 20160128 changelog](https://osu.ppy.sh/home/changelog/cuttingedge/20160128)
+[^mapset-restructure-proposal]: [Forum thread by ztrot (2016-02-13) "Mapset and Spread Restructure (Proposed)"](https://osu.ppy.sh/community/forums/topics/420223)
+[^meet-ephemeral]: ["Meet Ephemeral!" on ppy blog](https://blog.ppy.sh/post/139478794378/meet-ephemeral)
+[^mv2-timeline-doc]: ["Timeline of Events Surrounding Recent Changes" on Google Docs](https://docs.google.com/document/d/1VlFUIte8ho4tssRCucSBt96nTVAhsCLfJZcFVgzHjuk/edit)
```

Comment:
we'll definitely need to ask around for who wrote it and when. blame doesn't provide any hints (@MilkyIQ)

```suggestion
[^mv2-timeline-doc]: ["Timeline of Events Surrounding Recent Changes" on Google Docs](https://docs.google.com/document/d/1VlFUIte8ho4tssRCucSBt96nTVAhsCLfJZcFVgzHjuk/edit)<!-- TODO: find out author and date -->
```

Reply:
it's from https://blog.ppy.sh/post/139478794378/meet-ephemeral, so I think Ephemeral

Reply:
yes. [`8fd77dd` (#8093)](https://github.com/ppy/osu-wiki/pull/8093/commits/8fd77dd01f5847cc90428dd1782d8b9a19ac5a24)